src/Nri/Ui/Modal/V11.elm

```diff
@@ -438,7 +439,7 @@ viewModal config styles =
                     toContentAndFooter
                         { onlyFocusableElement =
                             List.map (Attrs.map config.wrapMsg)
-                                [ Key.onKeyDown
+                                [ onKeyDownPreventDefault
```

Comment:
I was seeing some weird behavior on the add-a-passage branch where the wrapping would correctly take me to the titleEditor, but then something would cause my focus to move to the URL bar. (Which, since focus then incorrectly moves under the modal rather than on to it, really breaks the experience). I'm not actually sure that this is the fix though, and I'm happy to walk it back and do more verification work before making a change like htis

Reply:
I don't think this change will hurt so I'm fine with merging as it is. Maybe doing the symlink dance to verify locally before doing the release dance can save us from some more dancing in the future, but up to you!

Reply:
Thanks!